### DISCUSSION TITLE
Web Components created using Stencil loads in Angular Application while it doesn't load for Storybook because shadow-root is not showing up

### DISCUSSION THREAD

**Discussion by 22f3001152** [storybookjs/storybook#25095] [Help] [answered]
### Summary

![Screenshot 2023-12-03 at 9 34 27 PM](https://github.com/storybookjs/storybook/assets/19172561/ede35f2d-f276-4db9-848c-affde322f723)

I am trying to load a simple webcomponent for angular. It works well on sample angular app. However when I load it on storybook it doesnt give me any console error. But it doesn't show the shadow root unlike the actual application. 

Attaching the screen shot. Please share your thoughts.

**Comment by valentinpalkovic** [comment]
Can you provide a simple reproduction?

**Reply by 22f3001152** [reply]
Sorry for the delay in responding. Unfortunately, I moved to a different functionality and took time for me get back. 

The reason why it was not loading is because I forgot to include `defineCustomElements` due to which the web-components were not loading. All looks good now.

**Comment by break-stuff** [comment, marked as answer]
Are you importing your compiled components from the "loader" directory? Here's a simple demo.

https://github.com/break-stuff/stencil-storybook-demo
